features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/BusinessInformation/BusinessInformation.jsx

```diff
@@ -148,6 +151,7 @@ class BusinessInformation extends React.Component {
                         <Paper className={classes.paperRoot} elevation={1}>
                             <FormControl margin='normal' className={classes.FormControlOdd}>
                                 <TextField
+                                    disabled={this.isNotCreator && this.isNotPublisher}
```

Comment:
If none of them(creator or publisher) can change this value ? then who can change it

Reply:
It is actually "Disabled for Non-creators AND Non-publishers === Enabled for Creators OR Publishers" some weird mathematics :D